### DISCUSSION TITLE
Navigation guards for refresh

### DISCUSSION THREAD

**Discussion by veryphatic** [directus/directus#24935] [Feature Requests]
### Describe the Improvement

I'm collecting data using a tablet, the overall experience is great, but some of my team are loosing their data when they accidently  pull down to reload (or hit refresh in a desktop client). It usually occurs after a side panel opens up and they're trying to close it (for example using a related dataset or repeater), or when using accordion to scroll back up to the correct position. 

The navigation guards for the back button work fine, but I'm wondering if it could be expanded for page reloads as well?

**Comment by izucken** [comment]
Hey @veryphatic are you still having this issue? Could you see if this suggestion https://github.com/directus/directus/discussions/22677 is relevant to you? It seems to me like we could solve both problems that way.

**Comment by veryphatic** [comment]
HI @iZucken yes I still experience this issue. I have worked around this by applying `html, body { overscroll-behavior: none; }` to the page to prevent the pull down to refresh action found on tablet devices.
